internal/cmd/login.go

```diff
@@ -73,7 +72,15 @@ func login(cmd *cobra.Command, envURL *url.URL) error {
 		fmt.Printf("Your browser has been opened to visit:\n\n\t%s\n\n", authURL.String())
 	}
 
-	token := readLine("Paste token here: ", cmd.InOrStdin())
+	fmt.Print("Paste token here: ")
+	var token string
+	scanner := bufio.NewScanner(cmd.InOrStdin())
+	scanner.Scan()
```

Comment:
suggest using `_ = scanner.Scan()` to make it clear that we're ignoring the return value intentionally